org.eclipse.transformer.jakarta/src/main/resources/org/eclipse/transformer/jakarta/jakarta-direct.properties

```diff
@@ -498,6 +498,43 @@ javax.persistence.spi.PeristenceProvider=jakarta.persistence.spi.PeristenceProvi
 javax.persistence.spi\:\:=jakarta.persistence.spi::
 # Other
 http\://xmlns.jcp.org/xml/ns/persistence=https://jakarta.ee/xml/ns/persistence
+javax.persistence.bean.manager=jakarta.persistence.bean.manager
```

Comment:
Where are these property names defined in the Jakarta EE API/specs? I did extensive analysis of the Jakarta EE API jars to locate String constants for inclusion in these rules and included all the String constants I found. These were not present.

Reply:
I found your spec references in the issue. Please add a comment above your new rules to document where they are sources. for example:

```properties
# Jakarta Persistence string constants from https://download.oracle.com/otn-pub/jcp/persistence-2_2-mrel-spec/JavaPersistence.pdf
```